packages/plugins/RSS3/src/SNSAdaptor/components/FeedCard/TokenTransferCard.tsx

```diff
@@ -0,0 +1,123 @@
+import { Image } from '@masknet/shared'
+import { makeStyles } from '@masknet/theme'
+import { RSS3BaseAPI } from '@masknet/web3-providers'
+import { isSameAddress } from '@masknet/web3-shared-base'
+import { Typography } from '@mui/material'
+import type { FC } from 'react'
+import { useI18N } from '../../../locales'
+import { Translate } from '../../../locales/i18n_generated'
+import { useFeedOwner } from '../../contexts'
+import { useAddressLabel } from '../../hooks'
+import { CardType } from '../share'
+import { CardFrame, FeedCardProps } from '../base'
+import { formatValue, Label } from './common'
+
+const useStyles = makeStyles<void, 'tokenIcon' | 'verboseToken'>()((theme, _, refs) => ({
+    summary: {
+        fontSize: '14px',
```

Comment:
The default font size of `<Typography />` is `14px`. So why do we still need to write it literally?

Reply:
OK. done.